assets/semi-structured/clickhouse/docker-build.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -e
+set -u
+
+script_dir="$(cd "$(dirname "${BASH_SOURCE[0]}")" &>/dev/null && pwd)"
+container_name="clickhouse-clp-bench"
+
+docker build \
+    -t "$container_name" \
+    "$script_dir" \
+    --file "$script_dir"/Dockerfile
```

Comment:
```suggestion
    --file "$script_dir/Dockerfile"
```

Let's quote entire paths rather than just the variable part of a path--it's simpler and it's nicer to read.